### DISCUSSION TITLE
Is there a way to exclude specific files?

### DISCUSSION THREAD

**Discussion by scarfur** [tadashi-aikawa/obsidian-various-complements-plugin#279] [Q&A] [answered]
As the title says :) 
If not, please consider this a feature request.

many thanks!

**Comment by tadashi-aikawa** [comment, marked as answer]
Hi, @scarfur :)

This setting could potentially assist you.

https://tadashi-aikawa.github.io/docs-obsidian-various-complements-plugin/4.%20Options/4.7.%20Internal%20link%20complement/%E2%9A%99%EF%B8%8F%20Exclude%20prefix%20path%20patterns%20%28Internal%20link%20complement%29/

**Reply by scarfur** [reply]
Thank you very much (works after app restart). Peace to you!
